content/docs/tasks/traffic-management/secure-ingress-sds/index.md

```diff
@@ -0,0 +1,488 @@
+
+1.  Start the `helloworld-v1` sample
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: helloworld-v1
+      labels:
+        app: helloworld-v1
+    spec:
+      ports:
+      - name: http
+        port: 8000
+      selector:
+        app: helloworld-v1
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: helloworld-v1
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: helloworld-v1
+        spec:
+          containers:
+          - name: helloworld
+            image: istio/examples-helloworld-v1
+            resources:
+              requests:
+                cpu: "100m"
+            imagePullPolicy: IfNotPresent #Always
+            ports:
+            - containerPort: 5000
+    EOF
+    {{< /text >}}
+
+1.  Create secret for ingress gateway. Assume you already create secret `httpbin-credential`. Now create secret `helloworld-credential`.
+
+    {{< text bash >}}
+    $ pushd mtls-go-example
+    $ ./generate.sh helloworld-v1.example.com <password>
+    $ mkdir ~+1/helloworld-v1.example.com && mv 1_root 2_intermediate 3_application 4_client ~+1/helloworld-v1.example.com
+    $ popd
+    $ kubectl create -n istio-system secret generic helloworld-credential --from-file=key=helloworld-v1.example.com/3_application/private/helloworld-v1.example.com.key.pem --from-file=cert=helloworld-v1.example.com/3_application/certs/helloworld-v1.example.com.cert.pem
+    {{< /text >}}
+
+1.  Define a Gateway with a server section for port 443.
+
+    Define a Gateway for two hosts, and specify `credentialName` to be `httpbin-credential` and `helloworld-credential` respectively.
+    TLS mode should be specified as SIMPLE. `serverCertificate` and `privateKey` should not be empty.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: mygateway
+    spec:
+      selector:
+        istio: ingressgateway # use istio default ingress gateway
+      servers:
+      - port:
+          number: 443
+          name: https-httpbin
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          serverCertificate: "use sds"
+          privateKey: "use sds"
+          credentialName: "httpbin-credential"
+        hosts:
+        - "httpbin.example.com"
+      - port:
+          number: 443
+          name: https-helloworld
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          serverCertificate: "use sds"
+          privateKey: "use sds"
+          credentialName: "helloworld-credential"
+        hosts:
+        - "helloworld-v1.example.com"
+    EOF
+    {{< /text >}}
+
+1.  Configure routes for traffic entering via the Gateway. Define the same `VirtualService`.
```

Comment:
Remove the period at the end, here and in other similar cases.

Reply:
Done.